### PR TITLE
fixed incorrect fake repo namespace

### PR DESCRIPTION
**Pull request by jrushlow**
fixes wrong fake repository class defined in bundle service

### CODE REVIEW

File: tests/IntegrationTests/ResetPasswordServiceDefinitionTest.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    public function bundleServiceDefinitionDataProvider(): \Generator
+    {
+        $prefix = 'symfonycasts.reset_password.';
+
+        yield [$prefix.'fake_request_repository'];
+        yield [$prefix.'cleaner'];
+        yield [$prefix.'random_generator'];
+        yield [$prefix.'token_generator'];
+        yield [$prefix.'helper'];
+        yield ['SymfonyCasts\Bundle\ResetPassword\Command\ResetPasswordRemoveExpiredCommand'];
```
Comment (jrushlow) [inline tests/IntegrationTests/ResetPasswordServiceDefinitionTest.php]: opted not to use `sprintf()` to improve readability. vs `sprintf('%sfake_......)`

File: tests/IntegrationTests/ResetPasswordServiceDefinitionTest.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
```
Comment (jrushlow) [inline tests/IntegrationTests/ResetPasswordServiceDefinitionTest.php]: We could provide `yield [SymfonyCasts....::class]` but yielding as is replicates what is actually set in the service config.

**Comment by kbond** [comment]
This is the fix for #78 ?

**Comment by jrushlow** [comment]
> This is the fix for #78 ?

Yes, early on I had renamed to fake repo to line up with existing naming conventions and forgot to change the service definition.

Once we get this pulled in, it’ll make things happy over in the maker PR. As is, `bin/console` explodes...

**Comment by weaverryan** [comment]
Awesome - thanks!

**Pull request merged**